### PR TITLE
#3114 - add support for exporting to inchikey file format

### PR DESCRIPTION
**Pull request by StarlaStarla**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)
Autotests that failed in pipeline while run successfully in local env
image

image

## Check list
- [x] unit-tests written
- [x] e2e-tests written
- [x] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [x] reviewers are notified about the pull request

### CODE REVIEW

File: ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts-snapshots/Open-Ketcher-Toolbar-hiding-items-1-chromium-linux.png
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts-snapshots/Open-Ketcher-Toolbar-hiding-items-1-chromium-linux.png:1]: I propose to pull master. The problem with this screenshot is supposed to be resolved. I've added a function to automatically wait for Indigo to load and now indigo icons are supposed to be always active on screenshots.

File: ketcher-autotests/tests/Reagents/SMILES-Format/reagents-smiles-format.spec.ts [hunk 1]
```diff
@@ -97,20 +95,24 @@ test.describe('Reagents SMILES format', () => {
     },
   );
 
-  test(`Detection molecule as reagent
-  and write reagent information in "Extended SMILES" format in "Preview" tab`, async ({
-    page,
-  }) => {
-    /*
+  test.fixme(
```
Comment (Nitvex) [inline ketcher-autotests/tests/Reagents/SMILES-Format/reagents-smiles-format.spec.ts:98]: If you mark some test with `fixme`, then I propose to add a comment – may be it is not working at all, or it is very unstable or some other reason

**Comment by Zhirnoff** [comment]
Please read comment in task
2023-08-30_11h34_17
2023-08-30_11h35_29

**Comment by Zhirnoff** [comment]
Pre-tested.
2023-08-30_11h43_28

**Review by Zhirnoff — APPROVED** [review]

File: packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.tsx [hunk 2]
```diff
@@ -129,7 +129,11 @@ const LeftToolbar = (props: Props) => {
       className={clsx(classes.root, className)}
       ref={ref}
     >
-      <div className={classes.buttons} ref={scrollRef}>
+      <div
+        className={classes.buttons}
+        ref={scrollRef}
+        data-testid="left-toolbar-buttons"
```
[the remark this answers is not in the document]
Comment (StarlaStarla) [reply packages/ketcher-react/src/script/ui/views/toolbars/LeftToolbar/LeftToolbar.tsx:135]: but line 128 in this file has already a element with data-testid="left-toolbar"

**Review by Nitvex — APPROVED** [review]

**Pull request merged**